db/migrate/20210208151828_add_activity_timestamp_to_tag.rb

```diff
@@ -0,0 +1,5 @@
+class AddActivityTimestampToTag < ActiveRecord::Migration[5.2]
+  def change
+    add_column :term_data, :activity_timestamp, :datetime
```

Comment:
OK, so to get this working for comments as well, we would have to add updaters to each of the different triggers for nodes, wikis, and comments, right?

Reply:
Yes.. I do it here for comments https://github.com/publiclab/plots2/pull/9165/files#diff-d521fcf96c3de2b8c810e0e8da2c12824cd1de1a7999f8b600284e8f358703c1R671

and here for nodes https://github.com/publiclab/plots2/pull/9165/files#diff-d521fcf96c3de2b8c810e0e8da2c12824cd1de1a7999f8b600284e8f358703c1R671

I only do it for creation... I only need to add the update one for Wikis...

Reply:
Awesome Ruth, as i said you are several steps ahead of me! I'm glad I understood this one -- thanks for explaining!